Clients/src/application/tools/extractToken.ts

```
@@ -12,7 +12,7 @@ export const extractUserToken = (
       console.error("Invalid token format");
       return null;
     }
-
+debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)